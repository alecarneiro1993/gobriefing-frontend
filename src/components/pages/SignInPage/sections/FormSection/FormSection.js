import React from "react";
import { useTranslation } from "react-i18next";

import { Container, Row, Column, SairaTypography } from "custom_modules";
import { CENTER, FULL, SECONDARY } from "utils/constants";

import Form from "./Form";

export default function FormSection() {
  const { t } = useTranslation();
  return (
    <Container height={`${FULL}%`} alignContent={CENTER}>
      <Row justify={CENTER}>
        <Column xs={4} align={CENTER}>
          <SairaTypography
            gutterBottom
            component="h1"
            variant="h4"
            color={SECONDARY}
          >
            {t("pages.sign_in.heading")}
          </SairaTypography>
        </Column>
      </Row>
      <Row>
        <Column xs>
          <Form />
        </Column>
      </Row>
    </Container>
  );
}
