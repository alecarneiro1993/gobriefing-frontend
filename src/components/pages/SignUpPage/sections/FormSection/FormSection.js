import React from 'react';
import { useTranslation } from 'react-i18next';

import { Container, Row, Column, RobotoTypography } from 'custom_modules';
import { CENTER, FULL, PRIMARY, OVERLINE } from 'utils/constants';

import SignUpForm from './SignUpForm';

function FormSection() {
  const { t } = useTranslation();
  return (
    <Container height={`${FULL}%`} alignContent={CENTER}>
      <Row justify={CENTER}>
        <Column xs={4} align={CENTER}>
          <RobotoTypography
            gutterBottom
            component="h1"
            variant={OVERLINE}
            size={1.2}
            color={PRIMARY}
          >
            {t('pages.sign_up.heading')}
          </RobotoTypography>
        </Column>
      </Row>
      <Row>
        <Column xs>
          <SignUpForm />
        </Column>
      </Row>
    </Container>
  );
}

export default FormSection;
