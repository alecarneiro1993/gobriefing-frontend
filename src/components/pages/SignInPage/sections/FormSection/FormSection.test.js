import React from "react";
import { shallow } from "enzyme";

import FormSection from "./FormSection";

describe("FormSection", () => {
  it("renders without crashing", () => {
    const wrapper = shallow(<FormSection />);
    expect(wrapper).toMatchSnapshot();
  });
});
