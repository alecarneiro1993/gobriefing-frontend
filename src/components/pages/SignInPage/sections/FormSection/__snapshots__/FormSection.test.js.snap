// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormSection renders without crashing 1`] = `
<Component
  alignContent="center"
  height="100%"
>
  <Component
    justify="center"
  >
    <Component
      align="center"
      xs={4}
    >
      <Styled(WithStyles(ForwardRef(Typography)))
        color="secondary"
        component="h1"
        gutterBottom={true}
        variant="h4"
      >
        pages.sign_in.heading
      </Styled(WithStyles(ForwardRef(Typography)))>
    </Component>
  </Component>
  <Component>
    <Component
      xs={true}
    >
      <withI18nextTranslation(SignInForm) />
    </Component>
  </Component>
</Component>
`;
